.bazelrc

```diff
@@ -113,6 +114,7 @@ build:asan --copt -fsanitize=address
 build:asan --copt -DADDRESS_SANITIZER
 build:asan --copt -fno-omit-frame-pointer
 build:asan --linkopt -fsanitize=address
+build:asan --no//:jemalloc_flag
```

Comment:
Q: why not?

Reply:
can you maybe comment it to the code?

Reply:
Asan test will inject their own malloc and with jemalloc, it'll crash :(